helloextend-protection/helloextend-protection.php

```
@@ -215,7 +215,8 @@ function helloextend_render_documentation_page()
 function helloextend_protection_style()
 {
     // Register stylesheets
-	$lastmodtime= filemtime(HELLOEXTEND_PLUGIN_URL. 'css/helloextend.css');
+    $css_path = HELLOEXTEND_PLUGIN_DIR . 'css/helloextend.css';
```

PHPMD (2.15.0)

The variable $css_path is not named in camelCase. (Controversial Rules)

(CamelCaseVariableName)